docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -193,6 +200,22 @@ Once the pipeline is set up, perform a *Full Content Sync* of the connector. The
 +
 Ask something to the AI Assistant related with the indexed data.
 
+[discrete]
+[[obs-ai-search-connectors-semantic-text]]
+==== Use a `semantic_text` field type to create AI Assistant embeddings
+
+To create the embeddings needed by the AI Assistant using a {ref}/semantic-text.html[`semantic_text`] field type:
+
+. Open the previously created connector, and select the *Mappings* tab.
+. Select *Add field*.
+. Under *Field type*, select *Semantic text*.
+. Under *Reference field*, select the field you want to use for model inference.
```

Comment:
I'm not exactly sure what this field does, but this was the explanation in the UI. @sorenlouv can we be more specific as to what the user needs to provide?